lib/python/test/testsuite/Template/Data.py

```diff
@@ -0,0 +1,85 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2022 PIConGPU contributors
+Authors: Mika Soren Voss
+License: GPLv3+
+
+/ToDo: Description of the test case
+"""
+
+# /ToDo:
+# import sys
+# sys.path.insert(1, "../../")
+# import testsuite as ts
+
+# general information about the test
+title = "Title"
+author = "Author"
+
+# directory specified
+resultDirection = None
```

Comment:
This is a template with all the parameters in it. The templates from MI or ESKHI do not use all parameters to keep the test case as clear as possible. Pure copying of these test cases does not provide all the possibilities of the test suite.

Reply:
So I think this code is very useful, because this file should serve as the basis for further test cases. It is thus comparable to docs/tbg_macros.cfg. But main.py in testsuite/Template can be removed, because it is every time the same.